otx/algorithms/common/adapters/torch/dataloaders/samplers/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Samplers for imbalanced and incremental learning."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
```suggestion

```

Reply:
There are many files about mis-annotated licenses. I want to align them all together in other PR.